src/XrdPfc/XrdPfcConfiguration.cc

```diff
@@ -517,6 +520,8 @@ bool Cache::Config(const char *config_filename, const char *parameters)
       TRACE(Error, "Config() Unable to create an OSS object");
       return false;
    }
+   if (orig_runmode) myEnv->Put("oss.runmode", orig_runmode);
```

Comment:
Out of curiosity, why add the restoration of the previous runmode? In practice, this is actually an endpoint and there will be no additional oss launches. I'm not adverse to this; just wondering about the rational.